GPT2/README.md

```diff
@@ -0,0 +1,13 @@
+# Runway GPT-2 model with Processing
+
+## Get Started
+1. Open Runway, add GPT-2 model to your workspace
+2. Select "Network" as input and ouput, Run the model
```

Comment:
There's no need to select Network as Input or Output. The network capabilities are always ON.